### PR TITLE
Improve branding a1

### LINKED ISSUE #22: Branding Page in Darkmode
**Describe the bug**
When eneabling darkmode, the first logo on the branding Page is barely visible

**To Reproduce**
1. Go to '/branding'
2. Enable Darkmode

**Expected behavior**
The background of the logo should make it easy to see it, even with darkmode.
Unbenannt

**Desktop (please complete the following information):**
  - OS: [Win10]
  - Browser: [Chrome]
  - Version: [Version 81.0.4044.138]

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the new behavior (if this is a feature change)?**
- Fix dark mode "light card" bug (Assignees: @Aichnerc ) #3 
  + Behaviour before Pull Request:
Currently when switching to dark mode, the `light card` background is dark. Closes #22 
  + Behaviour after Pull Request:
The `light card` remains the white background even in dark mode. Closes #22 

**Other information**:
- The version has been updated +0.0.1 (patch).

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
LGTM 🛹

**Review by LucaAllmaier — APPROVED** [review]
Looks good.

**Review by pinterid — APPROVED** [review]
Looks good.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

File: src/components/pages/BrandingPage/index.jsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ class BrandingPage extends React.Component {
                       <MDBIcon icon="download" className="pr-2" />
                       .png
                     </MDBBtn>
-                  </div>
+                    </div>*/}
```
Comment (schettn) [inline src/components/pages/BrandingPage/index.jsx]: Nah. Not the correct formatting

**Review by schettn — CHANGES_REQUESTED** [review]
Nearly

**Review by schettn — APPROVED** [review]
LGTM!

**Pull request merged**